skbio/stats/ordination/__init__.py

```diff
@@ -113,12 +113,13 @@
 from ._correspondence_analysis import CA
 from ._redundancy_analysis import RDA
 from ._canonical_correspondence_analysis import CCA
-from ._principal_coordinate_analysis import PCoA
+from ._principal_coordinate_analysis import PCoA, PCoABase
```

Comment:
That seems reasonable, another option is to bump `_principle_coordinate_analysis` to public so that it may be imported there. This feels like a better solution, though I am not sure why. Any thoughts @jairideout?